### DISCUSSION TITLE
Welcome

### DISCUSSION THREAD

**Discussion by StephenHillenbrand** [MarkW06071969/Tafe202206#1] [General]
Hi all, glad to be part of the team

**Comment by StephenHillenbrand** [comment]
@MarkW06071969 How you going with the local build for the menu?

**Comment by MarkW06071969** [comment]
Hi Stephen, am going well so far. Hope to have it uploaded to GitHub tonight or tomorrow morning at the latest.

Sent from Yahoo Mail on Android 
 
  On Sun, 28 Aug 2022 at 3:14 pm, ***@***.***> wrote:   


@MarkW06071969 How you going with the local build for the menu?

**Comment by MarkW06071969** [comment]
Hi Stephen, I have finished creating the development branch and the changes ( the main menu) in my local repository.
I am going to try and push to the GitHub repository in the morning. I have had trouble with that in the exercises.
Anyway, we can discuss this tomorrow.
See you then! 👍😁

Sent from Yahoo Mail on Android 
 
  On Sun, 28 Aug 2022 at 6:26 pm, Mark ***@***.***> wrote:   Hi Stephen, am going well so far. Hope to have it uploaded to GitHub tonight or tomorrow morning at the latest.

Sent from Yahoo Mail on Android 
 
  On Sun, 28 Aug 2022 at 3:14 pm, ***@***.***> wrote:   


@MarkW06071969 How you going with the local build for the menu?

**Comment by MarkW06071969** [comment]
Hi Stephen, the development branch has been cloned to GitHub.

Sent from Yahoo Mail on Android 
 
  On Sun, 28 Aug 2022 at 9:33 pm, Mark ***@***.***> wrote:   Hi Stephen, I have finished creating the development branch and the changes ( the main menu) in my local repository.
I am going to try and push to the GitHub repository in the morning. I have had trouble with that in the exercises.
Anyway, we can discuss this tomorrow.
See you then! 👍😁

Sent from Yahoo Mail on Android 
 
  On Sun, 28 Aug 2022 at 6:26 pm, Mark ***@***.***> wrote:   Hi Stephen, am going well so far. Hope to have it uploaded to GitHub tonight or tomorrow morning at the latest.

Sent from Yahoo Mail on Android 
 
  On Sun, 28 Aug 2022 at 3:14 pm, ***@***.***> wrote:   


@MarkW06071969 How you going with the local build for the menu?

**Reply by StephenHillenbrand** [reply]
Great to hear Mark. I'll clone a copy when I get to TAFE this morning. Hopefully it wasn't to painful.

**Comment by StephenHillenbrand** [comment]
Welcome @000961892 Hamish to the group.

**Comment by MarkW06071969** [comment]
https://tafesaedu-my.sharepoint.com/:v:/g/personal/camilo_carreno_student_tafesa_edu_au/EcqMhH127kxJn4x3MN55m3gBGChRE0QjxoJ6ikQsWiq70g
